### ISSUE TITLE
bug: Insane number of log files (1.1 MILLION files! Send help!)

### ISSUE THREAD

**Issue by hugalafutro** [FoxxMD/multi-scrobbler#234]
### Please check existing knowledge before opening an issue

- [X] I have [checked the FAQ](https://foxxmd.github.io/multi-scrobbler/docs/FAQ/) and [documentation](https://foxxmd.github.io/multi-scrobbler/docs/configuration/)
- [X] I have [searched through existing issues](https://github.com/FoxxMD/multi-scrobbler/issues?q=sort%3Aupdated-desc) and [discussions.](https://github.com/FoxxMD/multi-scrobbler/discussions)

### Describe the Bug

```
sammael in 🌐 docker-vm in multi-scrobbler/config/logs took 17s 
❯ ls | wc -l
1130815
```

### Platform

Docker

### Versions

- multiscrobbler 0.8.8
- docker 27.3.1

### Logs

```shell
Nothing in logs, not an reproducible issue. I could append output of `ls -la` if you'd really want :D
```

### Additional Context

So I noticed my borg backup takes over an hour and then I noticed there is a MILLION log files in multiscrobbler.

Perhaps log rotation could be implemented?

**Comment by hugalafutro** [comment]
they're mostly all empty as well:
![image](https://github.com/user-attachments/assets/8791fcea-0151-417f-b5f9-648c35854c19)

**Comment by FoxxMD** [comment]
Judging by the date on those files its [this DST bug](https://github.com/FoxxMD/multi-scrobbler/issues/214#issuecomment-2440156834) that was fixed in 0.8.7. My guess is you are in Europe due to DST occurring on Oct 27?  Euro users encountered the bug and then it should have been fixed for US countries in 0.8.7 before DST the following week.

You can safely delete all the files and it should not occur again.
